src/pages/webinars.js

```diff
@@ -0,0 +1,174 @@
+          <Link className={classes.bootcampLink} to="/bootcamp">
+            Learn about our Social Media Bootcamp →
+          </Link>
+        </div>
+        <WebinarCardGrid className={classes.grid}>
+          {webinars.map(({ node: webinar }) => {
+            // TODO: What's the proper way to do this with static gen?
+            // Is there a way to do it in the graphql query? Do we need to kick
+            // off a build daily to update this page?
+            const day = new Date(webinar.day);
+            const now = new Date();
+            if (day < now) {
+              return null;
+            }
+            return <WebinarCard key={webinar.id} webinar={webinar} />;
+          })}
+        </WebinarCardGrid>
+        <div className={classes.pastContainer}>
+          <div className={classes.past}>Past Webinars</div>
+          <div className={classes.pastDesc}>
+            Updated upon the completion of the weekly webinars
+          </div>
+          <a
+            href={"https://www.gotostage.com/channel/elmstreettechnology"}
+            target="_blank"
+            rel="noopener noreferrer"
+            className={classes.pastLink}
+          >
+            View Recordings
+          </a>
+        </div>
+      </Container>
+      <div className={classes.backgroundSlice}>
+        <svg
+          className={classes.backgroundTopSlice}
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 100 100"
+          preserveAspectRatio="none"
+        >
+          <polygon fill="white" points="0,100 100,0 100,100" />
+        </svg>
+        <svg
+          className={classes.backgroundBottomSlice}
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 100 100"
+          preserveAspectRatio="none"
+        >
+          <polygon fill="white" points="0,100 100,0 100,100" />
+        </svg>
+      </div>
+    </div>
+  );
+};
+
+export const query = graphql`
+  query webinarQuery {
+    allContentfulWebinar(limit: 1000, sort: { fields: [day], order: ASC }) {
+      edges {
+        node {
+          day
+          description
+          id
+          registrationUrl
+          time
+          title
+        }
+      }
+    }
+  }
+`;
+
+export default withStyles((theme) => ({
+  root: {
+    position: "relative",
+    paddingTop: "96px",
+    paddingBottom: "192px",
+  },
+  top: {
+    display: "flex",
+    flexWrap: "wrap",
+    alignItems: "center",
+  },
+  heading: {
+    fontSize: "32px",
+    fontWeight: "700",
+    marginRight: "auto",
+  },
+  bootcampLink: {
+    color: "inherit",
+    textDecoration: "none",
+    padding: "12px 0",
+
+    "&:hover": {
+      textDecoration: "underline",
+    },
+  },
+  grid: {
+    paddingTop: "96px",
+    paddingBottom: "120px",
+  },
+  backgroundSlice: {
+    position: "absolute",
+    top: "240px",
+    bottom: "0",
+    right: "0",
+    left: "0",
+    width: "100%",
+    height: "400px",
+    backgroundImage: `url('${Hexagons}')`,
```

Comment:
Thoughts on using a different pattern for the background slice on webinars vs. events?